ssds/storage.py

```diff
@@ -128,6 +124,15 @@ def __exit__(self, *args, **kwargs):
         for _ in self._async_set.consume():
             pass
 
+def _download_to(src_blob: AnyBlob, dst_blob: LocalBlob) -> Dict[str, str]:
+    src_blob.download(dst_blob.url)
+    return dict()
```

Comment:
Seems odd to return this?

Reply:
Returning `None` has side effects at another location in the code, unfortunately.

I'll add a comment.